src/SchemaGenerator.ts

```diff
@@ -0,0 +1,50 @@
+import {Node,rowClassify} from "./headerClassifier";
+
+
+export function generateSchema(header:Array<string>):object {
+    let headerList:Array<string> = [];
+    let schem:object = {};
```

Comment:
scheme

Reply:
### Scheme se traduce a plan, Schema es una representación de información o una concepción general.
fuente: https://english.stackexchange.com/questions/40702/difference-between-scheme-and-schema

Reply:
cambio los schem a schema